language_service/src/definition/tests.rs

```diff
@@ -38,3 +22,327 @@ fn assert_definition(source_with_markers: &str) {
     };
     assert_eq!(&expected_definition, &actual_definition);
 }
+
+#[test]
+fn callable() {
+    assert_definition(
```

Comment:
I have some local changes that would allow us to use `expect!` here (which probably would have been helpful to you before you made these changes, sorry). I can put those up in a PR soon.